server/app/index.js

```diff
@@ -9,9 +9,12 @@ const app = express();
 app.use(morgan('dev'));
 app.use(helmet());
 app.use(compression());
+app.use(require('./sessionMiddleware'));
 
 app.use(express.static(path.join(__dirname, '..', '..', 'public')));
 
+app.use(require('./sessionMiddleware'));
```

Comment:
This is just a clarification question, but why are you using app.use(require('./sessionMiddleware')); twice? Does it have a specific function?